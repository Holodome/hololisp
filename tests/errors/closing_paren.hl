(* 8 (- 0 8))
        (+ (* 1 123) 0
