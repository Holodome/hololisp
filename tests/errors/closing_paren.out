cli:2:9: error: Missing closing paren when reading list (eof encountered)
        (+ (* 1 123) 0
        ^
1 error generated.
